### PR TITLE
Getting started: add troubleshooting page

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Getting started: add troubleshooting page
- on onboarding/ installation issues
- partially addresses feedback from #33838
- move troubleshooting page from `docs` folder to `getting started`

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added a "Troubleshooting onboarding" link to the navigation on the "Getting Started" page for easier access to support resources.
	- Introduced comprehensive troubleshooting guides for installation issues, providing clear, actionable steps to help users resolve common problems.

- **Bug Fixes**
	- Removed outdated redirect entries for troubleshooting topics, streamlining user navigation to relevant documentation.

### CODE REVIEW

File: source/getting-started/troubleshooting.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+When trying to access Home Assistant in the browser, the browser shows the message “This site can’t be reached”.
+
+#### Description
+
+This means the browser can’t find your Home Assistant installation on the network.
+
+#### Resolution
+
+To resolve this issue, try the following steps:
```
Comment (agners) [inline source/getting-started/troubleshooting.markdown]: Good practice is going from physical layer up, I'd reorder:

3. -> 4. -> 5. -> 2. -> 1.

File: source/getting-started/troubleshooting.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+1. The system might still be starting up. Wait for a couple of minutes and refresh the browser.
+2. Make sure you typed the address correctly.
+   - Typically, this is `http://homeassistant.local:8123/`.
+   - If you are running an older Windows version or have a stricter network configuration, you might need to use `homeassistant:8123`.
```
Comment (agners) [inline source/getting-started/troubleshooting.markdown]: I'd make those two clickable URLs.

File: source/getting-started/troubleshooting.markdown [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+3. Make sure your Home Assistant device is powered up (LEDs are on).
+4. Make sure your Home Assistant installation is connected to the internet:
+   - Make sure the Ethernet cable is plugged-in to both Home Assistant and to your router or switch.
+   - Make sure your network is up and running and your router has internet access.
+5. Make sure the system on which you opened the browser to access Home Assistant is connected to the same network as Home Assistant.
+   - For example, you might have 2 networks with the same name, but one is on 2.4&nbsp;GHz and the other on 5&nbsp;GHz.
+   - One of your devices might default to another network. Make sure they are in the exact same network.
+6. Check your router's web interface to see what IP address is assigned to your Home Assistant installation.
+   - Enter this IP address (http://x.x.x.x:8123) directly into your browser.
```
Comment (agners) [inline source/getting-started/troubleshooting.markdown]: But this in single quotes, as the user needs to edit/type manually.

File: source/getting-started/troubleshooting.markdown [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+7. If you still can’t reach Home Assistant, connect keyboard and monitor to the device Home Assistant is running on to access the console and see where the Home Assistant gets stuck.
+   - If you are using a Home Assistant Green, follow these steps [to access the console](https://green.home-assistant.io/guides/use-terminal/).
+   - If you are using a Home Assistant Yellow, follow these steps [to access the console from Windows](https://yellow.home-assistant.io/guides/use-serial-console-windows/), or [to access the console from Linux or macOS](https://yellow.home-assistant.io/guides/use-serial-console-linux-macos/).
+8. [Reach out to our community for help](https://www.home-assistant.io/help/).
+   - If you report an issue, follow the steps on [reporting issues](/help/reporting_issues/).
```
Comment (agners) [inline source/getting-started/troubleshooting.markdown]: Let's not add report issue here. It is very unlikely a bug, as we have 100's of installation daily. This is an environment issue which needs to be investigated...

File: source/getting-started/troubleshooting.markdown [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+When trying to access Home Assistant in the browser, the browser message includes the error code “ERR_CONNECTION_REFUSED”.
+
+#### Description
+
+If you are using Chrome or Edge and ERR_CONNECTION_REFUSED is displayed, there might be a typo in the URL.
+
+#### Resolution
+
+1. Make sure you typed the address correctly.
```
Comment (agners) [inline source/getting-started/troubleshooting.markdown]:
```suggestion
1. Make sure you typed the address correctly. Especially, double check the port number is appended (`:8123`).
```

File: source/getting-started/troubleshooting.markdown [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+   - Typically, this is `http://homeassistant.local:8123/`.
+   - If you are running an older Windows version or have a stricter network configuration, you might need to use `homeassistant:8123`.
+2. Check the port number:
+   - Double check that you use :8123 at the end of the URL.
```
Comment (agners) [inline source/getting-started/troubleshooting.markdown]:
```suggestion
```

File: source/getting-started/troubleshooting.markdown [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+3. The system might still be starting up. Wait for a couple of minutes and refresh the browser.
+4. If you still get this error, then you might need to reinstall again.
+5. If you still can’t reach Home Assistant, connect a display to the device Home Assistant is running on to access the console to see where the Home Assistant gets stuck.
+   - If you are using a Home Assistant Green, follow these steps [to access the console](https://green.home-assistant.io/guides/use-terminal/).
+   - If you are using a Home Assistant Yellow, follow these steps [to access the console from Windows](https://yellow.home-assistant.io/guides/use-serial-console-windows/), or [to access the console from Linux or macOS](https://yellow.home-assistant.io/guides/use-serial-console-linux-macos/).
+6. [Reach out to our community for help](https://www.home-assistant.io/help/).
+   - If you report an issue, follow the steps on [reporting issues](/help/reporting_issues/).
```
Comment (agners) [inline source/getting-started/troubleshooting.markdown]: Same here, let's drop the issue report hint.

File: source/getting-started/troubleshooting.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/getting-started/troubleshooting.markdown]: yes, that would be a very systematic approach. But I think steps 1 and 2 are more likely to be the cause? I was trying to get the most likely causes out of the way first. also, If you put number one at it end there, you might as well not mention it at all. By the time you walked through all the previous steps, this should no longer be an issue.

File: source/installation/troubleshooting.markdown [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+This means the browser can’t find your Home Assistant installation on the network.
+
+### Resolution
+
+To resolve this issue, try the following steps:
+
+1. Make sure your Home Assistant device is powered up (LEDs are on).
+2. Make sure your Home Assistant installation is connected to the internet:
+   - Make sure the Ethernet cable is plugged-in to both Home Assistant and to your router or switch.
+   - Make sure your network is up and running and your router has internet access.
```
Comment (agners) [inline source/installation/troubleshooting.markdown]: Hm, "network is up and running" sounds  bit weird. Just this?

```suggestion
   - Make sure your network has internet access.
```

File: source/installation/troubleshooting.markdown [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+     - During installation, Home Assistant needs access to the following URLs:
+       - version.home-assistant.io: for online check and to fetch new version information.
+       - github.com: to update metadata for the add-on store.
+       - ghcr.io: the GitHub container registry to fetch new Home Assistant updates.
```
Comment (agners) [inline source/installation/troubleshooting.markdown]: We could also mention NTP here:

```suggestion
     - During first startup, time will be synchronized. Make sure the NTP is allowed in your network.
     - During first startup Home Assistant completes the installation. It needs access to the following URLs:
       - version.home-assistant.io: for online check and to fetch new version information.
       - github.com: to update metadata for the add-on store.
       - ghcr.io: the GitHub container registry to fetch new Home Assistant updates.
```

File: source/installation/troubleshooting.markdown [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+3. Make sure the system on which you opened the browser to access Home Assistant is connected to the same network as Home Assistant.
+   - For example, you might have 2 networks with the same name, but one is on 2.4&nbsp;GHz and the other on 5&nbsp;GHz.
+   - One of your devices might default to another network. Make sure they are in the exact same network.
```
Comment (agners) [inline source/installation/troubleshooting.markdown]: Well, typically, if SSID's are separated, they are still connected to the same network. It is rather uncommon that those would be different networks in that sense.

Also not sure what you meant by "One of your devices"....

How about simply checking WiFi, e.g.:
```
   - For example, if the system your Browser runs on is using WiFi, make sure it is using the WiFi which connects to the same network Home Assistant is connected to.
```

File: source/installation/troubleshooting.markdown [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+4. Make sure you typed the address correctly.
+   - Especially if the message includes the error code “ERR_CONNECTION_REFUSED”, it is likely that there was a typo in the URL.
```
Comment (agners) [inline source/installation/troubleshooting.markdown]:
```suggestion
   - Especially if the message includes the error code “ERR_CONNECTION_REFUSED”, it is likely that there was a typo in the port part of the URL (`:8123`).
```

**Review by agners — CHANGES_REQUESTED** [review]
Awesome work, this was much needed :star_struck:

**Review by agners — DISMISSED** [review]

**Review by agners — APPROVED** [review]

**Pull request merged**